client/verta/tests/test_model_registry/test_model_version.py

```diff
@@ -858,6 +860,60 @@ def test_redact(self, client, client_2, organization, created_entities):
         admin_model_ver.delete()
 
 
+# TODO: combine with test_experimentrun/test_attributes.py::TestComplexAttributes
```

Comment:
This was almost exactly copied & pasted from the existing `ExperimentRun` tests. I really need to figure out how to parametrize tests that are common between `ExperimentRun` and `ModelVersion`

Reply:
Would it be appropriate to move `get_attribute` onto `DeployableEntity` and have this test exist in a TestDeployableEntity` class? Then you can parameterize with both types of entities to run the same test for both an experiment run and a model version. See [tests/monitoring/alerts/test_entities.py#L214](https://github.com/VertaAI/modeldb/blob/ae30afeaea972f8e56de74450bcd656159da0c7a/client/verta/tests/monitoring/alerts/test_entities.py#L214) for an example.

Reply:
(And if the implementations need to diverge between the two classes, we can introduce `get_attribute` as an abstract method on DeployableEntity of course...)